<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калибровка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QPushButton" name="saveCalibrationButton">
     <property name="text">
      <string>Сохранить в файл</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QPushButton" name="loadCalibrationButton">
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Калибровка для шкалы миллиметров(мм/град)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QDoubleSpinBox" name="angleToRKoefDenominatorBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>-99999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="9" colspan="5">
    <widget class="QLabel" name="picLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>settingsPic.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Длина ножки (мм) L2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Угол между ножками(град) a</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="wireLengthBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Длина основания (мм) L1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="legWidthBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Смещение Х (мм) dx</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="legLengthBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Диаметр ножки (мм) W</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="legsAngleBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Смещение У (мм) dy</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Координаты отображать зеркально</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Люфт (град) b</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="deltaXBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-99999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="angleToRKoefNumeratorBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>-99999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="luftBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="10">
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QDoubleSpinBox" name="deltaYBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999998.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="10" column="6" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="11">
    <widget class="QLabel" name="currentCalibrationFileNamLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Текущая калибровка</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
